rust_src/rustfmt.toml

```diff
@@ -26,28 +26,22 @@ max_width = 100
 merge_derives = true
 newline_style = "Unix"
 normalize_comments = false
```

Comment:
Are these still supported in some way?

Reply:
They are default already. The options are deleted.

Reply:
Good! Thanks.